client/client.go

```diff
@@ -47,6 +47,9 @@ type (
 	// Options are optional parameters for Client creation.
 	Options = internal.ClientOptions
 
+	// FeatureFlags define which breaking changes can be enabled for client
+	FeatureFlags = internal.FeatureFlags
+
```

Comment:
this diff is fairly large already, but tbh I wonder if it's worth saying "nope, we should stop doing this immediately", and doing the new type + cast-or-whatever.  IDEs and code-generators really don't play well with these internal-aliases...